libsrc/common/doesclrscr.s

```diff
@@ -0,0 +1,14 @@
+;
+; Christian Groessler, June-2016
+;
+; unsigned char doesclrscr(void);
+;
+; returns 0/1 if after program termination the screen isn't/is cleared
+;
+
+        .export  _doesclrscrafterexit
+
+_doesclrscrafterexit:
```

Comment:
Please create an alias instead of spending bytes:

` _doesclrscrafterexit := return0 `